src/sentry/workflow_engine/processors/delayed_workflow.py

```diff
@@ -646,7 +647,11 @@ def fire_actions_for_groups(
                     organization,
                 ):
                     for action in filtered_actions:
-                        action.trigger(workflow_event_data, detector)
+                        trigger_action.delay(
+                            **build_trigger_action_task_params(
+                                action, detector, workflow_event_data
+                            )
```

Comment:
nit: might be nice to pull this out as a variable then splat the variable here -- that is mostly for debugging purposes later.